### PR TITLE
 OHRM5X-1721: Define LDAP config user-employee mapping field

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: installer/Migration/V5_2_0/lang-string/admin.yaml [hunk 1]
```diff
@@ -203,3 +203,30 @@ langStrings:
...
+    value: 'Attribute field to use when searching user objects. Ex: objectClass=person'
+    unitId: user_search_filter_input_hint
+  -
+    value: 'Attribute field to use as a unique immutable identifier for user objects. This is used to track username changes. Ex: entryUUID, objectGUID'
+    unitId: user_unique_attribute_input_hint
+  -
+    value: 'Field in OrangeHRM'
+    unitId: field_in_orangehrm
+  -
+    value: 'Field in LDAP Directory'
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply installer/Migration/V5_2_0/lang-string/admin.yaml:216]: For the table header this is correct

File: installer/Migration/V5_2_0/lang-string/admin.yaml [hunk 2]
```diff
@@ -203,3 +203,30 @@ langStrings:
...
+    unitId: field_in_ldap_directory
+  -
+    value: 'Use this field as a employee / user mapping field'
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply installer/Migration/V5_2_0/lang-string/admin.yaml]: Fixed

File: installer/Migration/V5_2_0/lang-string/admin.yaml [hunk 3]
```diff
@@ -203,3 +203,30 @@ langStrings:
...
     value: 'Merge LDAP Users With Existing System Users'
     unitId: merge_ldap_users_with_existing_system_users
+  -
+    value: 'Attribute field to use when searching user objects. Ex: objectClass=person'
+    unitId: user_search_filter_input_hint
+  -
+    value: 'Attribute field to use as a unique immutable identifier for user objects. This is used to track username changes. Ex: entryUUID, objectGUID'
+    unitId: user_unique_attribute_input_hint
+  -
+    value: 'Field in OrangeHRM'
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply installer/Migration/V5_2_0/lang-string/admin.yaml:213]: For the table header this is correct

File: installer/Migration/V5_2_0/lang-string/general.yaml [hunk 4]
```diff
@@ -36,47 +15,11 @@ langStrings:
...
   -
-    value: 'Configurations'
-    unitId: configurations
+    value: 'Authentication'
+    unitId: authentication
   -
-    value: 'Only show accessible employees on leave for other users'
-    unitId: only_show_accessible_employees_on_leave_for_other_users
+    value: 'Search results'
+    unitId: search_results
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply installer/Migration/V5_2_0/lang-string/general.yaml]: Moved

File: orangehrm-quick-start-guide.html [hunk 5]
```diff
@@ -1,106 +1,112 @@
...
+        <a
+          href="https://opensourcehelp.zendesk.com/knowledge/articles/6187572000540/en-us?brand_id=360001951359"
+          target="_blank"
+        >
+          Prerequisites for installing in Linux
+        </a>
+      </li>
+      <li>
+        <a
+          href="https://opensourcehelp.zendesk.com/knowledge/articles/6187576427804/en-us?brand_id=360001951359"
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply orangehrm-quick-start-guide.html]: Fixed

File: orangehrm-quick-start-guide.html [hunk 6]
```diff
@@ -1,106 +1,112 @@
...
+      it at our
+      <a href="http://www.orangehrm.com/forum/" target="_blank">forum</a>.
+    </p>
+
+    <h2>Prerequisites for installing OrangeHRM <ins class="version"></ins></h2>
+
+    <ul>
+      <li>
+        <a
+          href="https://opensourcehelp.zendesk.com/knowledge/articles/6187572000540/en-us?brand_id=360001951359"
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply orangehrm-quick-start-guide.html]: Fixed

File: orangehrm-quick-start-guide.html [hunk 7]
```diff
@@ -1,106 +1,112 @@
...
+        <a
+          href="https://starterhelp.orangehrm.com/hc/en-us/articles/5295915003666-OrangeHRM-Starter-Installation-Guide"
+          target="_blank"
+        >
+          Installing OrangeHRM using web installer
+        </a>
+      </li>
+    </ul>
+  </body>
+  <script type="text/javascript" src="installer/guide/index.js"></script>
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply orangehrm-quick-start-guide.html]: Done

**Review by RajithaKumara — APPROVED** [review]

File: orangehrm-quick-start-guide.html [hunk 8]
```diff
@@ -1,106 +1,111 @@
...
+
+    <h2>Installing OrangeHRM <ins class="version"></ins></h2>
+
+    <ul>
+      <li>
+        <a
+          href="https://starterhelp.orangehrm.com/hc/en-us/articles/5295915003666-OrangeHRM-Starter-Installation-Guide"
+          target="_blank"
+        >
+          Installing OrangeHRM using web installer
```
Comment (RajithaKumara) [inline orangehrm-quick-start-guide.html:106]: Any idea to add upgrader link?
https://starterhelp.orangehrm.com/hc/en-us/articles/6937346912402-OrangeHRM-Starter-Upgrade-Guide-For-5x-versions-

Comment (Super-Chama) [reply orangehrm-quick-start-guide.html:106]: Not sure. btw this is quick start guide, upgrader is for advance users who have already running setup

File: src/client/src/orangehrmPimPlugin/pages/employee/EmployeeJob.vue [hunk 9]
```diff
@@ -20,8 +20,9 @@
 
 <template>
   <edit-employee-layout
-    :employee-id="empNumber"
     screen="job"
+    :employee-id="empNumber"
+    :max-file-size="maxFileSize"
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPimPlugin/pages/employee/EmployeeJob.vue:25]: Fix for #1457

**Pull request merged**